useful_extras/create_comparison.py

```diff
@@ -118,17 +118,17 @@ def recreate_single_figure(
                 errors = unyt.unyt_array(this_line_dict["scatter"], units=plot.y_units)
 
                 # Data points from the bins with too few data points
-                plot_additional = True
-                try:
-                    additional_x = unyt.unyt_array(this_line_dict["additional_points_x"], units=plot.x_units)
-                    additional_y = unyt.unyt_array(this_line_dict["additional_points_y"], units=plot.y_units)
-                except KeyError:
-                    plot_additional = False
+                additional_x = unyt.unyt_array(this_line_dict.get("additional_points_x", []),
+                                               units=plot.x_units)
+                additional_y = unyt.unyt_array(this_line_dict.get("additional_points_y", []),
+                                               units=plot.y_units)
 
                 if line.scatter == "none":
-                    ax.plot(centers, heights, label=name)
+                    (mpl_line,) = ax.plot(centers, heights, label=name)
+                    ax.scatter(additional_x, additional_y, s=5, c=mpl_line.get_color())
```

Comment:
Please don't manually specify `s`, this should be specified in the matplotlib stylesheet.

Reply:
Done